linera-summary/src/ci_runtime_comparison.rs

```diff
@@ -11,17 +11,39 @@ use tracing::warn;
 type WorkflowName = String;
 
 #[derive(Serialize)]
-struct WorkflowJobRuntimeComparison {
+pub struct WorkflowJobRuntimeComparison {
     name: String,
     workflow_name: String,
-    base_runtime: String,
-    pr_runtime: String,
-    runtime_difference_pct: String,
+    base_runtime: u64,
+    pr_runtime: u64,
+    runtime_difference_pct: f64,
+}
+
+impl WorkflowJobRuntimeComparison {
+    pub fn name(&self) -> &str {
+        &self.name
+    }
+
+    pub fn workflow_name(&self) -> &str {
+        &self.workflow_name
+    }
+
+    pub fn base_runtime(&self) -> u64 {
+        self.base_runtime
+    }
+
+    pub fn pr_runtime(&self) -> u64 {
+        self.pr_runtime
+    }
+
+    pub fn runtime_difference_pct(&self) -> f64 {
+        self.runtime_difference_pct
+    }
 }
 
 // The key is the name of the workflow, and the value is a list of per job comparisons.
 #[derive(Serialize)]
-pub struct CiRuntimeComparison(HashMap<WorkflowName, Vec<WorkflowJobRuntimeComparison>>);
+pub struct CiRuntimeComparison(pub HashMap<WorkflowName, Vec<WorkflowJobRuntimeComparison>>);
```

Comment:
Optional idea: we could sort the workflow results by name (will probably require updates to the import and to the `collect` below).

```suggestion
pub struct CiRuntimeComparison(pub BTreeMap<WorkflowName, Vec<WorkflowJobRuntimeComparison>>);
```